### PR TITLE
Updating ES to generate jks if keys do not already exist

### PR DESCRIPTION
**Pull request by ewolinetz**
A more complete version of this work.
Part of this effort may also be to leave the certs in pkcs12 format instead of creating JKS files.

### CODE REVIEW

File: elasticsearch/run.sh [hunk 1]
```diff
@@ -87,6 +88,100 @@ sg_role_prometheus:
...
+
+  info "Building required jks files and truststore"
+
+  # check for lack of admin.jks
+  if [[ ! -e $secret_dir/admin.jks ]]; then
+    build_jks_from_pem "admin" "admin-key" "admin-cert" "admin-ca"
+  fi
+
+  # check for elasticsearch.key and elasticsearch.crt
+  if [[ -e $secret_dir/elasticsearch.key && -e $secret_dir/elasticsearch.crt ]]; then
```
Comment (richm) [inline elasticsearch/run.sh]: you check for the existence of `admin.jks` before writing it - do you need to check for `elasticsearch.jks` and `logging-es.jks` before writing them?

Comment (ewolinetz) [reply elasticsearch/run.sh]: so my thought here was if we have the .key and .crt then that would mean we shouldn't have the jks as generated by ansible. There shouldn't be any harm in adding the jks check though... I'll do that.

**Comment by ewolinetz** [comment]
I want to wait on this change just a little bit -- I may end updating the pem crt and key file names to match what the cert signing service would provide

**Comment by t0ffel** [comment]
PEM format seem to be supported by SearchGuard https://docs.search-guard.com/v5/configuring-tls

**Comment by ewolinetz** [comment]
will update this to follow the pattern in https://github.com/openshift/origin-aggregated-logging/pull/1266

**Comment by ewolinetz** [comment]
@jcantrill Kibana is failing to roll out now?

**Comment by richm** [comment]
> @jcantrill Kibana is failing to roll out now?

Looks like it - and the CI logs/artifacts are all messed up, so we can't even figure out why?

**Pull request merged**